### PR TITLE
Add algorithmic art talk

### PR DESCRIPTION
**Pull request by Alterrien**

### CODE REVIEW

**Comment by MrChoclate** [comment]
Nice pic ! :D

File: content/talks/algorithmic-art.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+type: talk
+title: Art algorithmique
+date: 10/12/2015
+image: prompt_talks.png
+short: "Qu'est-ce que l'art algorithmique, et à quoi ça ressemble ?"
+replay_id:
+authors:
+  - terrienale
+description: "Qu'est-ce que l'art algorithmique, et à quoi ça ressemble ?"
```
Comment (MrChoclate) [inline content/talks/algorithmic-art.yaml]: Could you find another description?
Something like "Je vous présente mon travail sur l'art algorithmique..."

**Comment by Thooms** [comment]
@Kirogo what's up here? This looks like a quite trivial issue, better close it as soon as possible.

**Comment by AntoineGirard** [comment]
@Thooms I need to upload video on YouTube before merge

**Comment by Thooms** [comment]
@neeev when you're done reviewing (and merging if everything is correct) #34, please treat this.

**Pull request merged**